compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -2902,20 +2891,20 @@ OMR::CodeGenerator::alignBinaryBufferCursor()
 
 
 int32_t
-OMR::CodeGenerator::setEstimatedLocationsForSnippetLabels(int32_t estimatedSnippetStart, bool isWarm)
+OMR::CodeGenerator::setEstimatedLocationsForSnippetLabels(int32_t estimatedSnippetStart)
    {
    TR::Snippet *cursor;
 
    self()->setEstimatedSnippetStart(estimatedSnippetStart);
 
    if (self()->hasTargetAddressSnippets())
       {
-      estimatedSnippetStart = self()->setEstimatedLocationsForTargetAddressSnippetLabels(estimatedSnippetStart, isWarm);
+      estimatedSnippetStart = self()->setEstimatedLocationsForTargetAddressSnippetLabels(estimatedSnippetStart);
       }
 
    for (auto iterator = _snippetList.begin(); iterator != _snippetList.end(); ++iterator)
       {
-      if ((*iterator)->isWarmSnippet() == isWarm)
+      if ((*iterator)->isWarmSnippet() == 0)
```

Comment:
Assuming `isWarmSnippet` function returns a `bool` can this be changed to:

```
if (!(*iterator)->isWarmSnippet())
```

to avoid comparison between `bool` and `int`. Similarly elsewhere in this change.